cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -133,6 +133,38 @@ module postgreServer 'br/public:avm/res/db-for-postgre-sql/flexible-server:0.1.4
 }
 {{- end}}
 
+{{- if .DbMySql}}
+var mysqlDatabaseName = '{{ .DbMySql.DatabaseName }}'
+var mysqlDatabaseUser = '{{ .DbMySql.DatabaseUser }}'
+module mysqlServer 'br/public:avm/res/db-for-my-sql/flexible-server:0.6.1' = {
+  name: 'mysqlServer'
+  params: {
+    // Required parameters
+    name: '${abbrs.dBforMySQLServers}${resourceToken}'
+    skuName: 'Standard_B1ms'
+    tier: 'Burstable'
+    // Non-required parameters
+    administratorLogin: mysqlDatabaseUser
+    administratorLoginPassword: mysqlDatabasePassword
+    geoRedundantBackup: 'Disabled'
+    firewallRules: [
```

Comment:
I'm not sure why exactly, but when I tried this locally, I ran into some firewall rule issues:

![Uploading image.png…]()

Reply:
Was able to connect after this tweak

Reply:
Updated